circle.yml

```diff
@@ -6,8 +6,37 @@ jobs:
       - image: quay.io/cbgeo/lbmdem
     steps:
       - checkout
-      # Configure CMake and run
+      # GCC 6.3
       - run:
-          name: Build LBM
-          command: mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++ -DCMAKE_EXPORT_COMPILE_COMMANDS=On .. && make clean && make -j8 && ./lbmdemtest
+          name: GCC-6 build
+          command: |
+            mkdir build && cd build
```

Comment:
suggest for all the steps that require an empty build directory
```
mkdir -p build
[ "$(ls -A build)" ] && rm -rf build/*
cd build
```

would make the steps work regardless of the order in which they are run